features/ccx-notification-service/customer_notifications.feature

```diff
@@ -1,35 +1,49 @@
 Feature: Customer Notifications
 
+  Background: Kafka is empty
+    Given Kafka topic "platform.notifications.ingress" is empty
+      And CCX Notification database is created for user postgres with password postgres
+      And CCX Notification database is empty 
+      And insights-content service is available on localhost:8082
 
   Scenario: Check that notification service does not need kafka if database has no new report
     Given Postgres is running
-      And CCX Notification database is created for user postgres with password postgres
```

Comment:
does it work w/o DB at all?

Reply:
this step have been moved to the `Bakground`, so the database is created in each step. It was just a way of not duplicating the same line in every scenario

Reply:
yes sure, I meant if we have a test for "problematic DB" cases

Reply:
ah, don' think so :/